src/components/HStack/HStack.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { css, cva, cx, type RecipeVariant } from "../../../styled-system/css";
+import type { Spacing } from "../../tokens/spacing";
+
+type As =
+  | "div"
+  | "section"
+  | "article"
+  | "main"
+  | "aside"
+  | "header"
+  | "footer"
+  | "nav"
+  | "span";
+
+export interface HStackProps
+  extends React.HTMLAttributes<HTMLElement>,
+    Partial<RecipeVariant<typeof hstackVariants>> {
+  /** 자식 요소들 (필수) */
+  children: React.ReactNode;
+  /** 렌더링할 HTML 요소 */
+  as?: As;
+  /** 요소 간 간격 */
+  gap?: Spacing;
+}
+
+/**
+ * - `children` 속성을 통해서 자식 요소들을 전달할 수 있습니다.
+ * - `as` 속성을 통해서 렌더링할 HTML 요소를 지정할 수 있습니다. 기본값은 `div`입니다.
+ * - `role` 속성을 통해서 역할을 지정할 수 있습니다.
+ * - `justify` 속성을 통해서 가로 정렬 방식을 지정할 수 있습니다. `start`, `center`, `end`, `between`, `around` 중 선택 가능합니다.
+ * - `reversed` 속성을 통해서 가로 배치 방향을 지정할 수 있습니다. 기본값은 `false`입니다.
+ * - `gap` 속성을 통해서 요소 간 간격을 지정할 수 있습니다.
+ *
+ * ### 접근성(Accessibility) 안내
+ * - 이미 시맨틱 태그를 쓰면(as=`nav` | `main` | `aside` | `footer` 등) 중복 role 지정은 피하시길 바랍니다.
+ * - 접근성을 위해 기본적으로 적절한 HTML 시맨틱 요소를 사용하고 필요시 ARIA 속성을 활용하여 접근성을 향상시킵니다.
+ */
+export const HStack = ({
+  children,
+  as = "div",
+  justify,
+  reversed = false,
+  gap,
+  className,
+  ...rest
+}: HStackProps) => {
+  const Component = as;
+
+  return React.createElement(
+    Component,
+    {
+      className: cx(
+        hstackVariants({
+          justify,
+          reversed,
+        }),
+        css({ gap }),
+        className,
+      ),
+      ...rest,
+    },
+    children,
+  );
+};
+
+const hstackVariants = cva({
+  base: {
+    display: "flex",
+    alignItems: "center",
+  },
+  variants: {
+    justify: {
```

Comment:
VStack 컴포넌트와 속성명 네이밍을 일관성있게 가져가면 좋을 것 같습니다.

```suggestion
    align: {
```

혹시 놓치셨다면, 관련 논의 링크드립니다: https://github.com/DaleStudy/daleui/pull/488#discussion_r2345643172

Reply:
아, 착각했네요! 반영하겠습니다.